src/main/java/com/ning/http/client/multipart/MultipartUtils.java

```diff
@@ -95,7 +97,7 @@ public static MultipartBody newMultipartBody(List<Part> parts, FluentCaseInsensi
     }
 
     private static byte[] generateMultipartBoundary() {
-        Random rand = new Random();
+        Random rand = new SecureRandom();
```

Comment:
Not sure if creating brand new `SecureRandom` for every single invocation of `generateMultipartBoundary` is optimal. Or even needed.

If I understand correctly, this boundary only needs to be unique within some container. Does it need to be _cryptographically secure_?

Reply:
Let me look into that a little more.